### PR TITLE
Ruby on Rails Controllers lesson: Clearer param values for Post params explanation

### PR DESCRIPTION
**Pull request by irene4**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

This params hash example momentarily confused me because the lowercase string values seemed like quotations woven into the text.

Original line 106: '... e.g. `params[:post_title]` might be "test post" and `params[:post_body]` might be "body of post" etc and these you can access with no issues.'

I think "My Test Post" etc. would more clearly represent the hash values in this example.

**2. This PR:**

- Changes example hash values in the params section of the Ruby on Rails Controllers lesson.

**3. Additional Information:**

### CODE REVIEW

File: ruby_on_rails/rails_basics/controllers.md [hunk 1]
```diff
@@ -103,13 +103,13 @@ In the example above, we saw `... code here to set up a new @post based on form
 
 With the `params` hash!  It acts just like a normal Ruby hash and contains the parameters of the request, stored as `:key => value` pairs.  So how do we get the ID of the post we're asking for? `params[:id]`.  You can access any parameters this way which have "scalar values", e.g. strings, numbers, booleans, `nil`... anything that's "flat".  
 
-Some forms will submit every field as a top level scalar entry in the params hash, e.g. `params[:post_title]` might be "test post" and `params[:post_body]` might be "body of post" etc and these you can access with no issues.  You have control over this, as you'll learn in the lessons on forms.
+Some forms will submit every field as a top level scalar entry in the params hash, e.g. `params[:post_title]` might be "My Test Post" and `params[:post_body]` might be "Body of post!" etc and these you can access with no issues.  You have control over this, as you'll learn in the lessons on forms.
```
Comment (KevinMulhern) [inline ruby_on_rails/rails_basics/controllers.md:106]: To go a step further with making these clearer, we could add "title" to the example

```suggestion
Some forms will submit every field as a top level scalar entry in the params hash, e.g. `params[:post_title]` might be "My Test Post Title" and `params[:post_body]` might be "Body of post!" etc and these you can access with no issues.  You have control over this, as you'll learn in the lessons on forms.
```

File: ruby_on_rails/rails_basics/controllers.md [hunk 2]
```diff
@@ -103,13 +103,13 @@ In the example above, we saw `... code here to set up a new @post based on form
...
 
-In our example, we will assume that our `params[:post]` is giving us a hash of Post attributes like `{ :title => "test post", :body => "this post rocks", :author_id => "1"}`, which is exactly what `Post.new` is expecting.  Whether you get the parameters packaged up in a hash or all on the top level as individual attributes (like `params[:id]`), again, is up to you and how you create your form.  But know it's usually easier for you to handle receiving a nicely packaged hash of Post attributes in your controller.
+In our example, we will assume that our `params[:post]` is giving us a hash of Post attributes like `{ :title => "My Test Post", :body => "this post rocks!", :author_id => "1"}`, which is exactly what `Post.new` is expecting.  Whether you get the parameters packaged up in a hash or all on the top level as individual attributes (like `params[:id]`), again, is up to you and how you create your form.  But know it's usually easier for you to handle receiving a nicely packaged hash of Post attributes in your controller.
```
Comment (KevinMulhern) [inline ruby_on_rails/rails_basics/controllers.md:112]:
```suggestion
In our example, we will assume that our `params[:post]` is giving us a hash of Post attributes like `{ :title => "My Test Post Title", :body => "this post rocks!", :author_id => "1"}`, which is exactly what `Post.new` is expecting.  Whether you get the parameters packaged up in a hash or all on the top level as individual attributes (like `params[:id]`), again, is up to you and how you create your form.  But know it's usually easier for you to handle receiving a nicely packaged hash of Post attributes in your controller.
```

**Comment by irene4** [comment]
Committed your changes

**Review by KevinMulhern — APPROVED** [review]
Thanks for taking the time to contribute @irene4 💪

**Pull request merged**